app/upgrades/v1/testnet/finality_params.go

```diff
@@ -1,13 +1,17 @@
 package testnet
 
-// TODO Some default parameters. Consider how to switch those depending on network:
-// mainnet, testnet, devnet etc.
+// The finality parameters are in the upgrade just because his structure
+// had an update and it is possible to overwrite during the upgrade.
+// The finality activation height is when the FP need to have their
+// program ready to start send finality signatures and it could be
+// the same block height where the allow list is expired in this case
+// babylon block height: 26120
 const FinalityParamStr = `{
   "max_active_finality_providers": 100,
   "signed_blocks_window": 100,
   "finality_sig_timeout": 3,
   "min_signed_per_window": "0.1",
   "min_pub_rand": 100,
```

Comment:
we could enforce larger `min_pub_rand`  but it can be done in separate pr probably (which actually choses the params for testnet)

@gitferry what do you thing would be good parameters for jailing ?

Reply:
Agreed `min_pub_rand` can be larger, maybe `500`? Can last more than a hour

> @gitferry what do you thing would be good parameters for jailing ?

Looks good as long as `min_signed_per_window` and `signed_sig_timeout` are consistent with corresponding jailing params of validators cc @babylon-devops